internal/commands/config.go

```diff
@@ -9,15 +9,16 @@ import (
 	"github.com/buildpacks/pack/logging"
 )
 
-func NewConfigCommand(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+func NewConfigCommand(logger logging.Logger, cfg config.Config, cfgPath string, client PackClient) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "config",
 		Short: "Interact with Pack's configuration",
 		RunE:  nil,
 	}
 
-	cmd.AddCommand(trustedBuilder(logger, cfg, cfgPath))
+	cmd.AddCommand(ConfigTrustedBuilder(logger, cfg, cfgPath))
```

Comment:
Though not strictly necessary for this PR, I changed this to make it easier to test alone, and require fewer mocks created in the `trustedBuilder` tests.